examples/pligos/flavors/types.yaml

```diff
@@ -3,6 +3,7 @@ route:
   port: string
   containerPort: string
   protocol: string
+  path: string
```

Comment:
I'm not convinced that path should be part of the route, and if you have a very good explanation of why we need more than `/` in the ingresses (which could be the case to cater for multiple users), then paths should maybe be repeated string.

Reply:
what I usually do is have something like `/api/v1` reserved for the backend and `/` is used for the frontend. And I think it does make sense, because a path maps to a port on a pod, which, using this configuration, is a single route.